hocr-eval

```diff
@@ -104,27 +108,28 @@ def edit_distance(a, b, threshold=99999):
                 cij = 1
             d = min(distances[i - 1][j] + 1, distances[i][j - 1] + 1,
                     distances[i - 1][j - 1] + cij)
-            if d >= threshold: return d
+            if d >= threshold:
+                return d
             distances[i][j] = d
     return distances[m][n]
 
 
-#def remove_tex(text):
-#    text_file = os.popen("echo %s | detex " %(text))
-#    text_plain = text_file.read()
-#    text_file.close()
-#    return text_plain
+# def remove_tex(text):
+#     text_file = os.popen("echo %s | detex " %(text))
+#     text_plain = text_file.read()
+#     text_file.close()
+#     return text_plain
```

Comment:
Do we still need that disabled code or can it be removed?

Reply:
Good question. I don't know what exactly the intention with this code was, but we also have currently a meaningless function instead:
```py
def remove_tex(text):
    return text
```

Maybe delete these things but make a comment/reminder somewhere?